src/main/arduino

```diff
@@ -0,0 +1,26 @@
+#include <LiquidCrystal.h>
+
+LiquidCrystal lcd = LiquidCrystal(5, 6, 7, 8, 9, 10, 11);
+int counter = 0;
+const int maxlen = 17;
+char buff[maxlen];
+
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop(){
+
+    if (Serial.available()>0){
+        char data = Serial.read();
+
+        if(data != '\0'){
```

Comment:
If data length is bigger than maxlen, I think it occurs illegal memory access in buff, am I wrong?